.circleci/config.yml

```diff
@@ -53,8 +53,9 @@ references:
     DO_TF_WORKSPACE: dev
     RELEASE_NAME: codacy
     NAMESPACE: codacy
-    K8S_VERSION: "1.14"
-    NUM_NODES: "5"
+    ENVIRONMENT: dev
+    K8S_VERSION: 1.14
+    NUM_NODES: 5
```

Comment:
Is this used? You removed this from the makefile. Why was this removed?

Reply:
It was just converted from string to int

Reply:
Where is this used?

Reply:
This was used when we were trying to have separate clusters per environment. But that's no longer the case so I vote for removing K8S_VERSION. The terraform files will control which k8s version to use in the cluster.
NUM_NODES was used when we did not have an autoscaling pool. We can drop it as well.

Reply:
Ok.. I think it makes sense to have it there